types/searchRange.js

```diff
@@ -17,6 +17,7 @@ class SearchRange extends Serializable {
     if (!(end instanceof Date)) {
       throw new Error('end must be an instance of Date')
     }
+
```

Comment:
Doesn't this still need the new constructor parameter?

Reply:
We took it out of SearchRange and put it in SearchOrder